.circleci/config.yml

```diff
@@ -48,12 +48,12 @@ jobs:
       - run:
           name: FOSSA Init
           command: fossa init
-      - run:
-          name: FOSSA Analyze
-          command: fossa analyze
-      - run:
-          name: FOSSA Test
-          command: fossa test
+      # - run:
```

Comment:
What is the reason the disable these lines?

Reply:
This step was failing. I asked crew-bronn and they have a support ticket in with that vendor. In the meantime, I just bypassed them. More context: https://auth0.slack.com/archives/C03H01D14/p1563304234062500